<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelRailLineDialog</class>
 <widget class="QDialog" name="SelRailLineDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 85, 127);
background-color: rgb(170, 170, 255);</string>
  </property>
  <widget class="QComboBox" name="rlCBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>请选择线路表：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtnEnter">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>158</width>
     <height>58</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>158</width>
     <height>58</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>58</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtnCancel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>230</y>
     <width>158</width>
     <height>58</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>158</width>
     <height>58</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>58</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rdbSX">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 900 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>上行</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rdbXX">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 900 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>下行</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
